src/healthcheck.js

```diff
@@ -0,0 +1,18 @@
+var http = require("http");
+var options = {
+  timeout: 2000,
+};
+var url = "http://localhost:80/health";
+var request = http.request(url, options, (res) => {
+  console.log(`STATUS: ${res.statusCode}`);
+  if (res.statusCode == 200) {
```

Comment:
```suggestion
  if (res.statusCode === 200) {
```

Reply:
The type of the response [StatusCode](https://nodejs.org/api/http.html#responsestatuscode) is a number. IMO, `===` is not needed for comparing that number with `200`.